src/components/SearchResults/List/List.component.jsx

```diff
@@ -19,7 +20,7 @@ const ListComponent = ({className, searchTerm, noElementsCustomMessage}) => {
     return (
         <div className={className}>
             {
-                (searchResultsState.totalPages > 1) ? <ListPaginator /> : null
+                (totalPages > 1) ? <ListPaginator /> : null
```

Comment:
Maybe this can be modified to something like this. Sorry in advance if there is an error in the syntax.
```suggestion
     return (
     {
        parsedData.length > 0 ? 
        <div className={className}>
            <ListPaginator /> 
                // now that the paginator works correctly i think its okay if you leave it even if there is only 
                // one page but it is up to you
            <div className="list-cards">
                {
                    parsedData.map((data) => {
                        return (<ListCard key={data.id} data={data} />)
                    })
                }
            </div>
            <ListPaginator />
        </div>
        : 
        <h4 style={{textAlign: "center"}}>{noElementsCustomMessage}</h4>
     }
        
    )
```